src/lib/ngsi/StatusCode.cpp

```diff
@@ -124,6 +124,51 @@ std::string StatusCode::render(Format format, const std::string& indent, bool co
 
 /* ****************************************************************************
 *
+* StatusCode::renderV2 -
+*
+* For version 2 of the API, the unnecessary 'reasonPhrase' is removed.
+*/
+std::string StatusCode::renderV2(bool comma)
```

Comment:
It's simply the render used for version 2.
Version 1 supports XML and JSON and adds beautifying stuff.
Version 2 is pure JSON and no beautifying.
The higher level render method will choose whether to call it or not, depending on **code**.